exercises/raindrops/description.md

```diff
@@ -6,10 +6,10 @@
 
 ## Examples
 
-- 28's factors are 2, 4, **7**, 14.
+- 28's factors are 1, 2, 4, **7**, 14, 28.
   - In raindrop-speak, this would be a simple "Plong".
-- 30's factors are 2, **3**, **5**, 6, 15.
+- 30's factors are 1, 2, **3**, **5**, 6, 15, 30.
   - In raindrop-speak, this would be a "PlingPlang".
-- 34 only has two factors- 2 and 17.
+- 34 only has four factors: 1, 2, 17, and 34.
```

Comment:
1 and self are special cases.  The problem is such that you have a factor with a replacement, it matters not that there are other factors.  This problem is not about primes (the definition of a number that is divisible by one and itself only) but about finding out if a number has a certain factor, and making that replacement.  

The lists of numbers here seem to be least common denominators, and so are accurate as listed, I think.